test/Middleware/WhoopsErrorResponseGeneratorTest.php

```diff
@@ -142,8 +142,8 @@ public function testJsonContentTypeResponseWithJsonResponseHandler()
         $this->request->getParsedBody()->willReturn([]);
 
         $this->response->withHeader('Content-Type', 'application/json')->will([$this->response, 'reveal']);
-        $this->response->withStatus(StatusCode::STATUS_INTERNAL_SERVER_ERROR)->will([$this->response, 'reveal']);
-        $this->response->getStatusCode()->willReturn(StatusCode::STATUS_INTERNAL_SERVER_ERROR);
+        $this->response->withStatus(StatusCode::STATUS_IM_A_TEAPOT)->will([$this->response, 'reveal']);
+        $this->response->getStatusCode()->willReturn(StatusCode::STATUS_IM_A_TEAPOT);
```

Comment:
> Looks good, in general. I think we need an additional test for when $e has an exception code that falls elsewhere in the 400-599 range, to validate that the response status is set accordingly.

I might have misunderstood that... or you wanted to see changes to the default `RunTimeException` error code.